backend/app/scripts/migrate_passwords_to_bcrypt.py

```
@@ -0,0 +1,290 @@
+            "total_users": 0,
+            "migrated_users": 0,
+            "failed_users": 0,
+            "skipped_users": 0,
+            "start_time": None,
+            "end_time": None,
+        }
+
+    def get_users_needing_migration(self, session: Session, limit: int) -> list[User]:
+        """获取需要迁移的用户"""
+        statement = select(User).where(
+            User.is_active == True,
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `User.is_active:` for truth checks

Replace with `User.is_active`

(E712)